tests/e2e/utils/src/flows/with-rest-api.js

```diff
@@ -152,5 +153,31 @@ export const withRestApi = {
 				expect( response.value ).toBe( defaultSetting.value );
 			}
 		}
+	},
+	/**
+	 * Create a batch of orders with the given `statuses`
+	 * using the "Batch Create Order" API endpoint.
+	 * 
+	 * @param statuses Array of order statuses
+	 */
+	batchCreateOrders : async (statuses) => {
+		const defaultOrder = config.get('orders.basicPaidOrder');
```

Comment:
This utility function will be more useful to package users if `defaultOrder` is read in the test function and passed as a parameter. That way other users can pass whatever order data they want to the function.

Reply:
Hi @rrennick no worries. I already started working on this earlier this week. 

> That way other users can pass whatever order data they want to the function.

Makes sense. I'll have the orders created inside the test instead, and then pass it as an array (of orders) to `batchCreateOrders()`

Reply:
> I'll have the orders created inside the test instead,

I believe the best way to do this is to use the utility function `createSimpleOrder()` found in `tests/e2e/utils/src/components.js`. But `createSimpleOrder()`  is using the UI to create the order. So I'm tempted to refactor `createSimpleOrder()` in such a way that it would use the API instead.

However, I saw from 116-gh-woocommerce/woocommerce-quality#issuecomment-900589554 that @zhongruige is already updating some of the tests to use the API. I'll double check with him to see if he already included `createSimpleOrder()` in his updates so that we avoid conflicts.

Reply:
Sorry, never mind my comment above 😅 

I just realized that refactoring `createSimpleOrder()` might impact all the tests using it.

Reply:
Done. Fixed in https://github.com/woocommerce/woocommerce/pull/30184/commits/71fe61a594514fb45371a0a661bcaff226a06eec